<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>integrated</class>
 <widget class="QWidget" name="integrated">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1070</width>
    <height>652</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>integrated</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1301</width>
     <height>651</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Tab 1</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Tab 2</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Page</string>
    </attribute>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>961</width>
       <height>241</height>
      </rect>
     </property>
     <row>
      <property name="text">
       <string>Material 1</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>Material 4</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>Material 3</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>Material 4</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>Material 5</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>Element Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Isotopic
Abundance</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Atomic Mass</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Atomic Fraction</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Density</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Thickness</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Component</string>
      </property>
     </column>
     <item row="0" column="0">
      <property name="text">
       <string>0000</string>
      </property>
     </item>
     <item row="0" column="1">
      <property name="text">
       <string>0000</string>
      </property>
     </item>
     <item row="0" column="2">
      <property name="text">
       <string>0000</string>
      </property>
     </item>
     <item row="0" column="3">
      <property name="text">
       <string>0000</string>
      </property>
     </item>
     <item row="0" column="4">
      <property name="text">
       <string>0000</string>
      </property>
     </item>
     <item row="0" column="5">
      <property name="text">
       <string>0000</string>
      </property>
     </item>
     <item row="0" column="6">
      <property name="text">
       <string>0000</string>
      </property>
     </item>
     <item row="1" column="0">
      <property name="text">
       <string>0000</string>
      </property>
     </item>
     <item row="1" column="1">
      <property name="text">
       <string>0000</string>
      </property>
     </item>
     <item row="1" column="2">
      <property name="text">
       <string>0000</string>
      </property>
     </item>
     <item row="1" column="3">
      <property name="text">
       <string>0000</string>
      </property>
     </item>
     <item row="1" column="4">
      <property name="text">
       <string>0000</string>
      </property>
     </item>
     <item row="1" column="5">
      <property name="text">
       <string>0000</string>
      </property>
     </item>
     <item row="1" column="6">
      <property name="text">
       <string>0000</string>
      </property>
     </item>
     <item row="2" column="0">
      <property name="text">
       <string>0000</string>
      </property>
     </item>
     <item row="2" column="1">
      <property name="text">
       <string>0000</string>
      </property>
     </item>
     <item row="2" column="2">
      <property name="text">
       <string>0000</string>
      </property>
     </item>
     <item row="2" column="3">
      <property name="text">
       <string>0000</string>
      </property>
     </item>
     <item row="2" column="4">
      <property name="text">
       <string>0000</string>
      </property>
     </item>
     <item row="2" column="5">
      <property name="text">
       <string>0000</string>
      </property>
     </item>
     <item row="2" column="6">
      <property name="text">
       <string>0000</string>
      </property>
     </item>
     <item row="3" column="0">
      <property name="text">
       <string>0000</string>
      </property>
     </item>
     <item row="3" column="1">
      <property name="text">
       <string>0000</string>
      </property>
     </item>
     <item row="3" column="2">
      <property name="text">
       <string>0000</string>
      </property>
     </item>
     <item row="3" column="3">
      <property name="text">
       <string>0000</string>
      </property>
     </item>
     <item row="3" column="4">
      <property name="text">
       <string>0000</string>
      </property>
     </item>
     <item row="3" column="5">
      <property name="text">
       <string>0000</string>
      </property>
     </item>
     <item row="3" column="6">
      <property name="text">
       <string>0000</string>
      </property>
     </item>
     <item row="4" column="0">
      <property name="text">
       <string>0000</string>
      </property>
     </item>
     <item row="4" column="1">
      <property name="text">
       <string>0000</string>
      </property>
     </item>
     <item row="4" column="2">
      <property name="text">
       <string>0000</string>
      </property>
     </item>
     <item row="4" column="3">
      <property name="text">
       <string>0000</string>
      </property>
     </item>
     <item row="4" column="4">
      <property name="text">
       <string>0000</string>
      </property>
     </item>
     <item row="4" column="5">
      <property name="text">
       <string>0000</string>
      </property>
     </item>
     <item row="4" column="6">
      <property name="text">
       <string>0000</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>860</x>
       <y>330</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Done</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Page</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
